subnetactor/types.go

```diff
@@ -37,7 +37,7 @@ type Votes struct {
 
 // ConsensusType defines the types of consensus supported
 // by subnets.
-type ConsensusType int64
+type ConsensusType uint64
```

Comment:
We use this type and the protocols below at least in two repos/packages: Eudico/Lotus and here. Should we store it only in one package and import it?

Reply:
`IpcGatewayAddr` must be `IPCGatewayAddr`

Reply:
Fixed